app/controllers/HomeController.php

```diff
@@ -1,7 +1,16 @@
 <?php
 
+require_once 'models/PostModel.php';
```

Comment:
I think it's worth understanding how namespaces are used in PHP - using them in Magento is a requirement, and while using `require_once` statements will do the job, I'd love to see some re-factoring to use namespaces.

Reference article:  https://opensource.com/article/23/4/autoloading-namespaces-php